sdk/devcenter/test-resources.bicep

```diff
@@ -16,11 +16,12 @@ var defaultNetworkConnectionName = 'sdk-networkconnection-${uniqueString('networ
 var defaultNetworkConnection2Name = 'sdk-networkconnection2-${uniqueString('networkConnection', '2022-09-01-preview', baseName, resourceGroup().name)}'
 var defaultMarketplaceDefinition = 'sdk-devboxdefinition-${uniqueString('devboxdefinition', '2022-09-01-preview', baseName, resourceGroup().name)}'
 var defaultCatalogName = 'sdk-default-catalog'
+var defaultScheduleName = 'default'
```

Comment:
Looks like [net - devcenter - tests](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=3109625&view=logs&j=76b7f760-7857-5526-9942-d88b580eb83a&t=0b78fc45-a08b-5d1a-bcae-155a32f152dd) pipeline is still failing. Could you please fix the issue before merging this PR?